docs/releasenotes.md

```diff
@@ -5,6 +5,15 @@ title: Releasenotes
 
 # Release notes BAG Huidige bevragingen API
 
+## Versie 1.5.0 (22-12-2021)
+In deze release is de volgende functionaliteit gerealiseerd:
+
+- fuzzy-search op adressen toegevoegd
+  - endpoint /adressen/zoek is deprecated verklaard.
+  - de parameter "zoekresultaatIdentificatie" bij het endpoint /adressen is deprecated verklaard.
+  - alle schema-componenten voor Zoekresultaat zijn deprecated verklaard.
+  - endpoint /adressen is uitgebreid met een parameter "q" waarmee een zoekstring voor fuzzy-search op adressen kan worden opgegeven.
```

Comment:
Moet je niet beginnen met deze laatste? Dus positief benaderen: vertellen wat we hebben verbeterd. Daarna de minder belangrijke consequenties opsommen (deprecated functionaliteit).

Ik vraag me ook af of voor iedereen de term "fuzzy-search" duidelijk is. Dat is bovendien een best breed begrip dat wellicht veel meer suggereert dan nu geboden wordt. 
Zo iets?: endpoint /adressen is uitgebreid met een parameter "q" waarmee met vrij op te geven zoekwoorden adressen kunnen worden gezocht. Het is nu niet meer nodig om in twee stappen te zoeken met /adressen/zoek gevolgd door /adressen met de zoekresultaatIdentificatie.